Users/controllers/usersController.js

```diff
@@ -0,0 +1,63 @@
+import { usersService } from '../services';
+
+const findAllUsers = async (req, res) => {
+  try {
+    const users = await usersService.findAllUsers();
+    res.status(200).json({ users });
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+};
+
+const userSignUp = async (req, res) => {
+  try {
+    const { email, name, password, account, nickname, phone_number } = req.body;
+    await usersService.userSignUp(
+      email,
+      name,
+      password,
+      account,
+      nickname,
+      phone_number
+    );
+
+    res.status(201).json({
+      message: 'USER_CREATED_SUCCESSFULLY',
+    });
+  } catch (err) {
+    res.status(500).json({
+      message: err.message,
+    });
+  }
+};
+
+const userLogin = async (req, res) => {
+  try {
+    console.log('여긴 컨트롤러');
+    const { email, password } = req.body;
+
+    if (!email && !password) {
+      let error = new Error('PLZ_INSERT_REQUIRED_INFORMATION');
+      error.statusCode = 404;
```

Comment:
```suggestion
    if (!email && !password) {
      let error = new Error('REQUIRED_INFORMATION_REQUIRED');
      error.statusCode = 400;
```
- 에러메세지에 `PLZ`와 같은 줄임말은 사용하지 않으시는 것이 좋습니다 :)
- `404`는 서버측에 해당하는 데이터나 자원이 없을 때를 나타내는 에러 코드로, 지금과 같이 프론트 측에서 필요한 정보가 누락된 경우에는 `400, Bad request`가 더 적절합니다.

아래 다른 코드에서도 줄임말과 404 수정해주세요 👍

Reply:
수정 완료입니다 소헌님! 감사합니다. e14430f7be4d759d832322cde586678607cfa1e9